<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="0" failed="3" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[STEP: ENTERED LOGIN CREDENTIALS ]]>
    </line>
    <line>
      <![CDATA[STEP: ENTERED LOGIN CREDENTIALS ]]>
    </line>
    <line>
      <![CDATA[STEP: ENTERED LOGIN CREDENTIALS ]]>
    </line>
  </reporter-output>
  <suite started-at="2023-11-01T05:14:53 IST" name="Suite" finished-at="2023-11-01T05:15:20 IST" duration-ms="26947">
    <groups>
    </groups>
    <test started-at="2023-11-01T05:14:53 IST" name="Test_ChromeLatest_Win10" finished-at="2023-11-01T05:15:08 IST" duration-ms="14723">
      <class name="MyAccount.Tests.LoginTest">
        <test-method is-config="true" signature="setUpDriverAndLogin_bmConfig(java.lang.String, java.lang.String, java.lang.reflect.Method, org.testng.ITestResult)[pri:0, instance:MyAccount.Tests.LoginTest@5a63f509]" started-at="2023-11-01T05:14:53 IST" name="setUpDriverAndLogin_bmConfig" finished-at="2023-11-01T05:15:05 IST" duration-ms="12064" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[browserstack.config.json]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[public void MyAccount.Tests.LoginTest.loginTest() throws java.io.IOException]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[[TestResult name={null} status=CREATED method=LoginTest.loginTest()[pri:0, instance:MyAccount.Tests.LoginTest@5a63f509] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[STEP: ENTERED LOGIN CREDENTIALS ]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUpDriverAndLogin_bmConfig -->
        <test-method signature="loginTest()[pri:0, instance:MyAccount.Tests.LoginTest@5a63f509]" started-at="2023-11-01T05:15:05 IST" name="loginTest" finished-at="2023-11-01T05:15:05 IST" duration-ms="81" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.fail(Assert.java:102)
at MyAccount.Tests.LoginTest.loginTest(LoginTest.java:19)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:68)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="quitDriver_amConfig(java.lang.String, org.testng.ITestResult)[pri:0, instance:MyAccount.Tests.LoginTest@5a63f509]" started-at="2023-11-01T05:15:06 IST" name="quitDriver_amConfig" finished-at="2023-11-01T05:15:08 IST" duration-ms="1604" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[browserstack.config.json]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[TestResult name=loginTest status=FAILURE method=LoginTest.loginTest()[pri:0, instance:MyAccount.Tests.LoginTest@5a63f509] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver_amConfig -->
      </class> <!-- MyAccount.Tests.LoginTest -->
    </test> <!-- Test_ChromeLatest_Win10 -->
    <test started-at="2023-11-01T05:14:53 IST" name="Test_Edge117_Win10" finished-at="2023-11-01T05:15:10 IST" duration-ms="16719">
      <class name="MyAccount.Tests.LoginTest">
        <test-method is-config="true" signature="createExtentReport_bsConfig()[pri:0, instance:MyAccount.Tests.LoginTest@481a996b]" started-at="2023-11-01T05:14:52 IST" name="createExtentReport_bsConfig" finished-at="2023-11-01T05:14:53 IST" duration-ms="403" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createExtentReport_bsConfig -->
        <test-method is-config="true" signature="setUpDriverAndLogin_bmConfig(java.lang.String, java.lang.String, java.lang.reflect.Method, org.testng.ITestResult)[pri:0, instance:MyAccount.Tests.LoginTest@481a996b]" started-at="2023-11-01T05:14:53 IST" name="setUpDriverAndLogin_bmConfig" finished-at="2023-11-01T05:15:07 IST" duration-ms="14153" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[browserstack.config.json]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[edge]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[public void MyAccount.Tests.LoginTest.loginTest() throws java.io.IOException]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[[TestResult name={null} status=CREATED method=LoginTest.loginTest()[pri:0, instance:MyAccount.Tests.LoginTest@481a996b] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[STEP: ENTERED LOGIN CREDENTIALS ]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUpDriverAndLogin_bmConfig -->
        <test-method signature="loginTest()[pri:0, instance:MyAccount.Tests.LoginTest@481a996b]" started-at="2023-11-01T05:15:07 IST" name="loginTest" finished-at="2023-11-01T05:15:07 IST" duration-ms="14" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.fail(Assert.java:102)
at MyAccount.Tests.LoginTest.loginTest(LoginTest.java:19)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:68)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="quitDriver_amConfig(java.lang.String, org.testng.ITestResult)[pri:0, instance:MyAccount.Tests.LoginTest@481a996b]" started-at="2023-11-01T05:15:08 IST" name="quitDriver_amConfig" finished-at="2023-11-01T05:15:10 IST" duration-ms="1752" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[browserstack.config.json]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[TestResult name=loginTest status=FAILURE method=LoginTest.loginTest()[pri:0, instance:MyAccount.Tests.LoginTest@481a996b] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver_amConfig -->
        <test-method is-config="true" signature="flushExtentReport_asConfig()[pri:0, instance:MyAccount.Tests.LoginTest@481a996b]" started-at="2023-11-01T05:15:20 IST" name="flushExtentReport_asConfig" finished-at="2023-11-01T05:15:21 IST" duration-ms="1465" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flushExtentReport_asConfig -->
      </class> <!-- MyAccount.Tests.LoginTest -->
    </test> <!-- Test_Edge117_Win10 -->
    <test started-at="2023-11-01T05:14:53 IST" name="Test_Firefox118_Win11" finished-at="2023-11-01T05:15:20 IST" duration-ms="26947">
      <class name="MyAccount.Tests.LoginTest">
        <test-method is-config="true" signature="setUpDriverAndLogin_bmConfig(java.lang.String, java.lang.String, java.lang.reflect.Method, org.testng.ITestResult)[pri:0, instance:MyAccount.Tests.LoginTest@2145b572]" started-at="2023-11-01T05:14:53 IST" name="setUpDriverAndLogin_bmConfig" finished-at="2023-11-01T05:15:18 IST" duration-ms="24566" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[browserstack.config.json]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[public void MyAccount.Tests.LoginTest.loginTest() throws java.io.IOException]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[[TestResult name={null} status=CREATED method=LoginTest.loginTest()[pri:0, instance:MyAccount.Tests.LoginTest@2145b572] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[STEP: ENTERED LOGIN CREDENTIALS ]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUpDriverAndLogin_bmConfig -->
        <test-method signature="loginTest()[pri:0, instance:MyAccount.Tests.LoginTest@2145b572]" started-at="2023-11-01T05:15:18 IST" name="loginTest" finished-at="2023-11-01T05:15:18 IST" duration-ms="7" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.fail(Assert.java:102)
at MyAccount.Tests.LoginTest.loginTest(LoginTest.java:19)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:68)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="quitDriver_amConfig(java.lang.String, org.testng.ITestResult)[pri:0, instance:MyAccount.Tests.LoginTest@2145b572]" started-at="2023-11-01T05:15:18 IST" name="quitDriver_amConfig" finished-at="2023-11-01T05:15:20 IST" duration-ms="1691" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[browserstack.config.json]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[TestResult name=loginTest status=FAILURE method=LoginTest.loginTest()[pri:0, instance:MyAccount.Tests.LoginTest@2145b572] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitDriver_amConfig -->
      </class> <!-- MyAccount.Tests.LoginTest -->
    </test> <!-- Test_Firefox118_Win11 -->
  </suite> <!-- Suite -->
</testng-results>
